README.md

```diff
@@ -0,0 +1,83 @@
+# PyVista Tutorial
+
+## Abstract
+
+This tutorial will introduce 3D data visualization using PyVista (https://github.com/pyvista/pyvista).
+PyVista offers 3D plotting and mesh analysis through a streamlined interface for the Visualization Toolkit (VTK).
+PyVista uses VTK (http://www.vtk.org) under the covers and provides a very Python-friendly API.
+
+At the end of the tutorial users will be able to
+
+- Use PyVista effectively to create 3D visualizations. 
+- Understand the basic class structure of PyVista.
+
+## Keywords
+
+[visualization](https://github.com/topics/visualization)
+[meshviewer](https://github.com/topics/meshviewer)
+[vtk](https://github.com/topics/vtk)
+[3d](https://github.com/topics/3d)
+
+## Other Information and Files
+
+### Tutorial Description.
+Same as Abstract.
+
+### Tutorial Outline.
+
+0. Getting Started - Why PyVista? (10 min for talk, 10 min for exercise)
+
+1. Basic usage - Reading and plotting 3D data.  (10 min for talk, 10 min for exercise)
+
+2. What is a Mesh? - Learn about "Point", "Cell" and "Attributes". (10 min for talk, 10 min for exercise)
+
+3. PyVista Data Model - Learn about the data model of the PyVista framework. (10 min for talk, 5 min for exercise)
+
+4. Mesh Creation - Exercise to build and plot meshes in PyVista. (10 min for talk, 5 min for exercise)
+
+6. Charts API - Exercise to use the Charts API to plot your data in PyVista. (10 min for talk, 5 min for exercise)
```

Comment:
```suggestion
6. Mesh animation - Show how to animate meshes using the PyVista API. (10 min for talk, 5 min for exercise)
```

Reply:
Yeah. Mesh animation is more general for users.